x-pack/test/functional/services/ml/single_metric_viewer.ts

```diff
@@ -117,5 +117,89 @@ export function MachineLearningSingleMetricViewerProvider({ getService }: FtrPro
       await testSubjects.click('mlAnomalyResultsViewSelectorExplorer');
       await testSubjects.existOrFail('mlPageAnomalyExplorer');
     },
+
+    async openConfigForControl(entityFieldName: string) {
+      const isPopoverOpened = await testSubjects.exists(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+
+      if (isPopoverOpened) {
+        return;
+      }
+
+      await testSubjects.click(
+        `mlSingleMetricViewerEntitySelectionConfigButton_${entityFieldName}`
+      );
+      await testSubjects.existOrFail(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+    },
+
+    // TODO move to some common service for EUI components
+    async assertRadioGroupValue(testSubject: string, expectedValue: string) {
+      const sortByRadioGroup = await testSubjects.find(testSubject);
+      const input = await sortByRadioGroup.findByCssSelector(':checked');
+      const selectedSortBy = await input.getAttribute('id');
+      expect(selectedSortBy).to.eql(
+        expectedValue,
+        `Expected the radio group value to equal "${expectedValue}"`
+      );
+    },
+
+    // TODO move to some common service for EUI components
+    async selectRadioGroupValue(testSubject: string, value: string) {
+      const sortByRadioGroup = await testSubjects.find(testSubject);
```

Comment:
I think `sortByRadioGroup` should be named a bit more generic, maybe just `radioGroup`.